src/node.cc

```diff
@@ -1206,34 +1198,8 @@ Local<Value> MakeCallback(Environment* env,
     }
   }
 
-  if (ran_init_callback && !pre_fn.IsEmpty()) {
```

Comment:
How is `before` and `after` called now. The propused API is that `MakeCallback(..., async_id, ...)` will call `before` and `after` but this doesn't appear to do that.